scripts/load_configured_data

```diff
@@ -54,24 +54,35 @@ function load_configured_data() {
 
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
-    if [ "${1:-}" = "--help" ]
-    then
-        usage
-    else
-        configPath=${1:-config/config.xml}
-        if [ "${1:-}" = "--production" ]
-        then
-          # Make these environment variables available
-          # to docker-compose in production.
-          export COMPOSE_FILE="docker-compose.yml:docker-compose.production.yml"
-          export GIT_COMMIT
+    configPath="config/config.xml"
+    DATA_CONTAINERS=("postgres" "django")
+    for FLAG in "${@}"
+    do
+        echo "Processing flag ${FLAG}"
+        case "${FLAG}" in
+            --help|-h)
+                usage
+                exit
+                ;;
+            --production|-p)
+                # Make these environment variables available
+                # to docker-compose in production.
+                export COMPOSE_FILE="docker-compose.yml:docker-compose.production.yml"
+                export GIT_COMMIT
+                DATA_CONTAINERS=("django")
+                shift
```

Comment:
I don't think these `shift` statements are doing anything because this is a `for` loop rather than a `while` loop. I wrote two scripts:
```bash
#!/bin/bash

for FLAG in "${@}"
do
    echo "${FLAG}"
    shift
done
```
and
```bash
#!/bin/bash

for FLAG in "${@}"
do
    echo "${FLAG}"
done
```
Calling `./test.sh 1 2 3 4` prints back
```
1
2
3
4
```
in both cases.